<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointLayerWidget</class>
 <widget class="QWidget" name="JointLayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="layerLabel">
       <property name="text">
        <string>Object/Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover { background: #edf2f4;
}
QPushButton:focus { background: #edf2f4;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../resource.qrc">
         <normaloff>:/resource/icons/icon_dnd_area_remove.png</normaloff>:/resource/icons/icon_dnd_area_remove.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="layerComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
